### DISCUSSION TITLE
lightcurve binning vs moving average

### DISCUSSION THREAD

**Discussion by orionlee** [lightkurve/lightkurve#1187] [Q&A]
Hi, 

Historically, I used to create a moving average (e.g., over a 10 minute window) of a lightcurve's flux for visualization. I've started to use binning, as it seems to be the norm.

From science / statistics perspective, what'd be the difference (aside the fact that moving average does not downsample, but "smoothes out" the samples)?

Thanks!

**Comment by gully** [comment]
My hunch is that moving average would retain the size of the array whereas binning decreases the size of the array?  For applications where fewer samples are needed for plotting simplicity or computational speed than binning is better?  I may not have understood the question though!

**Reply by orionlee** [reply]
Moving average does indeed retain the size of the array, unlike binning which reduces the size.

In addition to cases gully mentioned (reduced size for practical computation limits, possibly better clarity in plot), I wonder if there are other differences one should be aware of (in interpreting and/or using the result). 

Thanks!
